src/gameobjects/units/cUnit.cpp

```diff
@@ -595,21 +595,32 @@ void cUnit::draw_health()
     if (r > 255) r = 255;
 
     // bar itself
-    renderDrawer->renderRectFillColor(drawx, drawy, width_x, height_y, 0, 0, 0,255);
-    renderDrawer->renderRectFillColor(drawx, drawy, (w - 1), height_y, (Uint8)r,(Uint8)g, 32,255);
+    renderDrawer->renderRectFillColor(drawx, drawy, width_x, height_y, 0, 0, 0,160);
+    renderDrawer->renderRectFillColor(drawx, drawy, (w - 1), height_y, (Uint8)r,(Uint8)g, 32,160);
 
     // bar around it (only when it makes sense due zooming)
     if (height_y > 2) {
-        renderDrawer->renderRectColor(drawx, drawy, width_x, height_y, 255, 255, 255,255);
+        renderDrawer->renderRectColor(drawx, drawy, width_x, height_y, 255, 255, 255,160);
     }
+}
 
+void cUnit::draw_group()
+{
+    if (iHitPoints < 0) return;
+
+    // draw units health
+    /// float width_x = mapCamera->factorZoomLevel(getBmpWidth());
+    /// int height_y = mapCamera->factorZoomLevel(4);
+    /// int drawx = draw_x();
+    /// int drawy = draw_y() - (height_y + 2);
     // draw group
-    if (iGroup > 0 &&
-            iPlayer == HUMAN) {
+    if (iGroup > 0 && iPlayer == HUMAN) {
+        // @mira I don't fix group name without acces to textDrawer: fixed on ctx branch
```

Comment:
can these comments be removed?

Reply:
// @mira  it an internal tag that i use to my work